config/default/jenkins-infra.yaml

```diff
@@ -0,0 +1,127 @@
+                            username: "${GITHUB_USERNAME}"
+                            password: "${GITHUB_PASSWORD}"
+                            scope: GLOBAL
+                        - string:
+                            scope: GLOBAL
+                            id: "updatecli-github-token"
+                            secret: '${UPDATECLI_GITHUB_TOKEN}'
+                            description: Github Token used by updatecli to update version
+                        - basicSSHUserPrivateKey:
+                            scope: GLOBAL
+                            id: "charts-secrets"
+                            username: ${SSH_CHARTS_SECRETS_USERNAME}
+                            #passphrase: ${SSH_CHARTS_SECRETS_PASSWORD}
+                            description: "SSH privkey used to access jenkins-infra/charts-secrets"
+                            privateKeySource:
+                              directEntry:
+                                privateKey: ${SSH_CHARTS_SECRETS_PRIVKEY}
+              k8s-settings: |
+                jenkins:
+                  clouds:
+                    - kubernetes:
+                        containerCapStr: "100"
+                        jenkinsTunnel: "default-jenkins-infra-agent:50000"
+                        jenkinsUrl: "http://default-jenkins-infra:8080"
+                        maxRequestsPerHostStr: "300"
+                        name: "kubernetes"
+                        namespace: "release"
+                        podRetention: "Never"
+                        serverUrl: "https://kubernetes.default"
+                        templates:
+                          - name: jnlp-linux
+                            nodeSelector: "kubernetes.io/os=linux"
+                            containers:
+                              - name: jnlp
+                                image: "jenkins/jnlp-slave:latest-jdk11"
+                                resourceLimitCpu: "500m"
+                                resourceLimitMemory: "512Mi"
+                                resourceRequestCpu: "500m"
+                                resourceRequestMemory: "512Mi"
+                                args: "^${computer.jnlpmac} ^${computer.name}"
+                                alwaysPullImage: true
+                          - name: jnlp-windows
+                            nodeSelector: "kubernetes.io/os=windows"
+                            containers:
+                              - name: jnlp
+                                image: "jenkins4eval/jnlp-agent:latest-windows"
+                                resourceLimitCpu: "500m"
+                                resourceLimitMemory: "512Mi"
+                                resourceRequestCpu: "500m"
+                                resourceRequestMemory: "512Mi"
+                                args: "^${computer.jnlpmac} ^${computer.name}"
+                                alwaysPullImage: true
+                            label: "jnlp-windows"
```

Comment:
Shouldn't the label also be on the Linux one?

Reply:
Nice catch, this label is not needed.